resources/files/windows/PuppetBolt/PuppetBolt.psm1

```diff
@@ -1,3 +1,5 @@
+. $PSScriptRoot\PuppetBoltFunctions.ps1
```

Comment:
There's no real point doing this.  You may as well just throw all of it into the PSM1 file.  Saves WIX stuff.